src/app/shared/components/category-navigation/category-navigation.component.ts

```diff
@@ -29,7 +29,7 @@ export class CategoryNavigationComponent {
   constructor(private router: Router) {}
 
   activeRoute(): string | undefined {
-    return this.router.url.split('/')[2];
+    return this.router.url.split('?')[0].split('/')[2];
```

Comment:
a może brać tylko drugi path ? Jak było wcześniej
this.router.url.split('/')[2],
/znajdz-pomoc/**noclegi**
/dodaj-ogloszenie/**noclegi**